portfolio.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<section class="navigationBar">
+		<div class="navigationButtons">
+			<div class="home">
```

Comment:
I don't think you need this extra div outside of your link tags as you can put the classes you've made directly onto the links and it will still work. However I'd actually suggest removing all the divs outside of the link tags and adding justify-content: space-between; to your navigationButtons class as this will result in less code.

Reply:
Have chaged the code to basically this, although I used justify-content: center with a margin cos I wanted the buttons all central.